lib/src/play_store_search_api.dart

```diff
@@ -99,15 +99,32 @@ class PlayStoreResults {
   /// Returns field releaseNotes from Play Store results. When there are no
   /// release notes, the main app description is used.
   static String? releaseNotes(Document response) {
+    RegExp releaseNotesSpan = RegExp(r'>(.*?)</span>');
     try {
       final sectionElements = response.getElementsByClassName('W4P4ne');
       final releaseNotesElement = sectionElements.firstWhere(
           (elm) => elm.querySelector('.wSaTQd')!.text == 'What\'s New',
           orElse: () => sectionElements[0]);
-      final releaseNotes = releaseNotesElement
+
+      String? releaseNotes;
+      
+      Element? rawReleaseNotes = releaseNotesElement
           .querySelector('.PHBdkd')
-          ?.querySelector('.DWPxHb')
-          ?.text;
+          ?.querySelector('.DWPxHb');
+      String? innerHtml = rawReleaseNotes!.innerHtml.toString();
```

Comment:
Using `rawReleaseNotes!` may cause an exception.

Reply:
Won't the try catch block be able to catch it?

I mean what kind of exception you are exactly thinking of?